nodejs/examples/rabbitmq.js

```diff
@@ -0,0 +1,30 @@
+const amqp = require('amqplib/callback_api');
+const config = require("platformsh-config").config();
+
+exports.usageExample = async function() {
+    try {
+        const credentials = config.credentials('rabbitmq');
+        console.log(credentials);
+
+        let connectionString = 'amqp://' + credentials.username + ':' + credentials.password + '@' + credentials.host + ':' + credentials.port;
+        console.log(connectionString);
+
+        let connection = await amqp.connect(connectionString);
+        console.log(connection);
+
+        let channel = await connection.createChannel();
+        console.log(channel);
+
+        let queue = 'hellotest';
+        let message = 'Deploy Friday!';
+
+        await channel.assertQueue(queue, {durable: false});
+        await channel.sendToQueue(queue, Buffer.from(message))
```

Comment:
`sendToQueue` behaves like node's `Stream.writable`: it returns a `boolean`, not a `Promise`. You don't have anything to `await` here.
You can ignore the returned value though.

JavaScript lets you write anything and try to make sense of it (with all the implications this can have...). That's why nothing complains about the fact that you're `await`ing for a primitive value when it would only make sense for a `Promise`.